index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="./css/common.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="./css/hamburger.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="./css/mobile.css">
+    <link rel="stylesheet" type="text/css" media="screen and (min-width: 747px)" href="./css/desktop.css">
+</head>
+<body>
+    <header>
+        <nav class="dk top-nav-container">
+            <article class="top-nav">
+                <ul class="social-icon">
+                    <li><img src="./assets/social-icon/facebook_f_64px.png" alt="facebook"></li>
+                    <li><img src="./assets/social-icon/twitter_64px.png" alt="twitter"></li>
+                </ul>
+                <ul class="h-link">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
```

Comment:
- [ ] 👏👏 Great job on adding all the top-nav items! However, after closely reviewing the design, I noticed that the icons and other items have different spacing compared to your current implementation. To align with the design requirements, please ensure that the icons and other items have consistent spacing by adjusting the margins or padding accordingly.

| yours | design |
| -- | -- |
|  ![image](https://github.com/sagieramos/CAPSTONE-ONE/assets/55840999/e1f13374-5ba0-4360-bd7c-4e96cef60b04) | ![image](https://github.com/sagieramos/CAPSTONE-ONE/assets/55840999/daa194a8-5391-437c-b2d8-bff138c3aa76) | 


By making these adjustments, your top-nav will look more polished and aligned with the design. Keep up the good work and happy coding! 🚀💻🌟